Week3/week3.txt

```diff
@@ -0,0 +1,209 @@
+cur.execute(table)
+con.commit()
+
+2. Insert 5 Records directly, 5 records using tuple and 5 records using taking input from user
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+ins="insert into employee\
+    values\
+    (101,'om','Account',25000,'surat'),\
+    (102,'sai','Inventory',28000,'vyara'),\
+    (103,'ram','IT',30000,'ghandhinagar'),\
+    (104,'pritam','HR',23000,'mumbai'),\
+    (105,'krishna','Inventory',25000,'baroda')"
+cur.execute(ins)
+con.commit()
+'''
++-----+---------+-----------+-------+--------------+
+| eid |  ename  |   dept    | basic |    branch    |
++-----+---------+-----------+-------+--------------+
+| 101 | om      | Account   | 25000 | surat        |
+| 102 | sai     | Inventory | 28000 | vyara        |
+| 103 | ram     | IT        | 30000 | ghandhinagar |
+| 104 | pritam  | HR        | 23000 | mumbai       |
+| 105 | krishna | Inventory | 25000 | baroda       |
++-----+---------+-----------+-------+--------------+
+'''
+
+#record input using tuple
+record= [
+        (106,'Krunal','HR',21000,'Bardoli'),
+        (107,'Radhe','Account',25000,'sarbhon'),
+        (108,'Rajesh','IT',26500,'Navsari'),
+        (109,'Dhruv','Inventory',20000,'Bardoli'),
+        (110,'Grijesh','HR',23000,'surat')
+        ]
+cur.executemany("insert into employee values(?,?,?,?,?)",record)
+con.commit()
+
+'''
++-----+---------+-----------+-------+---------+
+| eid |  ename  |   dept    | basic | branch  |
++-----+---------+-----------+-------+---------+
+| 106 | Krunal  | HR        | 21000 | Bardoli |
+| 107 | Radhe   | Account   | 25000 | sarbhon |
+| 108 | Rajesh  | IT        | 26500 | Navsari |
+| 109 | Dhruv   | Inventory | 20000 | Bardoli |
+| 110 | Grijesh | HR        | 23000 | surat   |
++-----+---------+-----------+-------+---------+
+'''
+
+#record insert using user input
+ins2="insert into employee values(?,?,?,?,?)"
+l=[ ] #create empty list
+for i in range(0,5):
+    n=int(input('Enter Employee Id: '))
+    name=input('Enter Employee Name: ')
+    dept=input('Enter Department Name: ')
+    basic=int(input('Enter Basic: '))
+    branch=input('Enter Branch: ')
+    t=[n,name,dept,basic,branch]
+    l.append(t)
+cur.executemany(ins2,l)
+con.commit()
+'''
++-----+---------+-----------+-------+--------+
+| eid |  ename  |   dept    | basic | branch |
++-----+---------+-----------+-------+--------+
+| 111 | John    | Inventory | 30000 | Vyara  |
+| 112 | Mohan   | Account   | 32000 | Delhi  |
+| 113 | Vikas   | HR        | 42000 | Pune   |
+| 114 | Vrijesh | IT        | 31000 | Mumbai |
+| 115 | Amit    | HR        | 41000 | Mumbai |
++-----+---------+-----------+-------+--------+
+'''
+
+3. Update records who are from "Surat" branch with increment in salary 10%.
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+q1=  "update employee set \
+    basic=basic+basic*10/100\
+    where branch is 'surat'\
+    "
+cur.execute(q1)
+con.commit()
+

```

Comment:
update records who  are from mumbai branch with increment is salary 24% and update dept post "account".

Reply:
update employee set \
    basic=basic+basic*24/100\
    where branch is 'mumbai'\